docs/source/restore/index.rst

```diff
@@ -6,162 +6,79 @@ Restore
    :hidden:
    :maxdepth: 2
 
-   download-files-command
+   restore-tables
+   restore-schema
+   examples
 
-The following procedure lets you restore data to a new cluster to get a cluster clone from snapshots stored in a backup location.
+.. contents::
+   :depth: 2
+   :local:
 
-**Prerequisites**
+The :ref:`sctool restore <sctool-restore>` command allows you to schedule a restore of backed-up data into a cluster.
```

Comment:
Why would one need to schadule a restore?
IMHO restore can be a ad-hoc command only (at least in docs)